src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -261,7 +263,7 @@ export default class ReviewCollapsibleChapter extends React.Component {
                 aria-controls={`collapsible-${this.id}`}
                 onClick={this.props.toggleButtonClicked}
               >
-                {chapterTitle || ''}
+                {`${this.state.editing ? 'Edit' : ''} ${chapterTitle || ''}`}
```

Comment:
Decide whether or not to prepend "Edit" to the button text based on the `editing` state.